common/types/shared.go

```diff
@@ -5572,6 +5688,50 @@ type SignalWithStartWorkflowExecutionRequest struct {
 	JitterStartSeconds                  *int32                 `json:"jitterStartSeconds,omitempty"`
 }
 
+func (v *SignalWithStartWorkflowExecutionRequest) BodyToString() string {
+	cloneRequest := SignalWithStartWorkflowExecutionRequest{
+		Domain:       v.Domain,
+		WorkflowID:   v.WorkflowID,
+		SignalName:   v.SignalName,
+		Identity:     v.Identity,
+		RequestID:    v.RequestID,
+		CronSchedule: v.CronSchedule,
+	}
+	// deep copy the value within pointers
```

Comment:
same here, explain why you are doing this manually

Reply:
done